package.json

```diff
@@ -29,12 +29,21 @@
     "build": "microbundle-crl --no-compress --format modern,cjs",
     "chromatic": "npx chromatic --project-token pq9dwlqeah",
     "lint": "eslint --ext ts,tsx src",
+    "precommit": "lint-staged",
     "prepare": "npm run build",
     "start": "microbundle-crl watch --no-compress --format modern,cjs",
     "storybook": "start-storybook -p 9009",
     "test": "jest",
     "types": "tsc --noEmit"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": ["npm run lint"]
```

Comment:
I don't think we want to run the whole lint task here... pretty sure that lint-staged can pass off _just_ the files that got staged to whatever command is here.

I know you can pass args to npm scripts by separating the flags with `--` (so, for example: `npm run thing -- someArg`). Maybe that's possible here? 

I would want to validate that the only files being linted are the files that have been staged, so that we don't lint the _whole_ project every time.